src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstSide + secondSide) * height) / 2;
```

Comment:
(firstParallel + secondParallel) / 2: integer division in floating-point context .
Please pay attention to [this](https://www.javatpoint.com/java-int-to-double) article.